### PR TITLE
Belching

### PR DESCRIPTION
**Pull request by BorisTakerman**
## About the PR
Adds a Belching/Burping emote to Dwarves, Humans, Oni, & Goblins.

## Why / Balance
Its just a small emote I would like to see in game. If you think it would fit other races it can be easily modified to allow them to, I just didn't know if other species except the near-humans would physiologically be able to.

## How to test
Open the game, play a Human, Dwarf, Oni, or Goblin. Open your Emote Wheel and select Vocal, and then burp away. You can test to see if other races cant by playing other races and attempting the same actions, or manually trying to emote by typing "belch", "burp", "burps", etc. 

## Media
https://youtu.be/LV5-bMR2LXI

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
- add: A New Belch Emote for Human, Dwarf, Goblin, Oni, and Harpy.

### CODE REVIEW

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 1]
```diff
@@ -270,6 +270,29 @@
...
+  chatMessages: ["chat-emote-msg-belch"]
+  chatTriggers:
+    - belch
+    - belch.
+    - burp
+    - burp.
+    - belchs
+    - belchs.
+    - burps
+    - burps.
```
Comment (dvir001) [inline Resources/Prototypes/Voice/speech_emotes.yml:294]: Add options with ``!``

**Comment by dvir001** [comment]
Please move everything under ``_NF``

**Comment by BorisTakerman** [comment]
> Please move everything (Unless its impossible like an edit to dwarf file) under `_NF` And add `# Frontier` comments to every single line that was added outside of `_NF`

I believe I have moved everything I could to _NF

**Review by whatston3 — APPROVED** [review]
Works alright.  belch_3 seems to be loud.  Might suggest reducing the volume of it a bit, otherwise, works fine.

**Comment by dvir001** [comment]
Herpy?

**Comment by BorisTakerman** [comment]
> Herpy?

Do you want me to allow Harpy's to do it?

**Comment by dvir001** [comment]
> > Herpy?
> 
> Do you want me to allow Harpy's to do it?

Yes please

**Comment by dvir001** [comment]
Note to myself look if it all working fine on upstream merge after.

**Pull request merged**